bentoml/_internal/bento/bento.py

```diff
@@ -88,23 +88,45 @@ def _Bento_set_fs(bento: "Bento", _: t.Any, new_fs: "FS") -> "FS":
 @attr.define(repr=False, auto_attribs=False)
 class Bento(StoreItem):
     _tag: Tag = attr.field()
-    _fs: "FS" = attr.field(on_setattr=_Bento_set_fs)
+    __fs: "FS" = attr.field(on_setattr=_Bento_set_fs)
```

Comment:
why we want this renaming for _fs while other private member still use single underscore?

Reply:
actually we shouldn't use double underscore

Reply:
The reason for the double underscore is that `_fs` is now a method.